dev/tasks/macros.jinja

```diff
@@ -184,15 +184,13 @@ env:
 {% endmacro %}
 
 {%- macro github_upload_wheel_scientific_python(pattern) -%}
-  {%- if arrow.is_default_branch() -%}
```

Comment:
I suppose you'll restore this condition before merging the PR?

Reply:
For sure! That's just to be able to test this here on the PR (I could maybe also just have merged it without testing, though)

Reply:
Reverted the temp commit